.github/workflows/manual-deploy-obscuro-gateway.yml

```
@@ -0,0 +1,67 @@
+# Deploys Obscuro Gateway on Azure for Testnet
+# Builds the Obscuro Gateway image, pushes the image to dockerhub and starts the Obscuro Gateway on Azure
+
+name: '[M] Deploy Obscuro Gateway Testnet'
+run-name: '[M] Deploy Obscuro Gateway Testnet ( ${{ github.event.inputs.testnet_type }} )'
+on:
+  workflow_dispatch:
+    inputs:
+      testnet_type:
+        description: 'Testnet Type'
+        required: true
+        default: 'dev-testnet'
+        type: choice
+        options:
+          - 'dev-testnet'
+          - 'testnet'
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: 'Set up Docker'
+        uses: docker/setup-buildx-action@v1
+
+      - name: 'Set Obscuro Gateway build tag for testnet'
+        if: ${{ github.event.inputs.testnet_type == 'testnet' }}
+        run: |
+          echo "OBSCURO_GATEWAY_BUILD_TAG=testnetobscuronet.azurecr.io/obscuronet/obscuro_gateway_testnet:latest" >> $GITHUB_ENV
+
+      - name: 'Set Obscuro Gateway build tag for dev-testnet'
+        if: ${{ github.event.inputs.testnet_type == 'dev-testnet' }}
+        run: |
+          echo "OBSCURO_GATEWAY_BUILD_TAG=testnetobscuronet.azurecr.io/obscuronet/dev_obscuro_gateway_testnet:latest" >> $GITHUB_ENV
+
+      - name: 'Login to Azure docker registry'
+        uses: azure/docker-login@v1
+        with:
+          login-server: testnetobscuronet.azurecr.io
+          username: ${{ secrets.REGISTRY_USERNAME }}
```

This was changed due to output secret clashes.

```suggestion
          username: testnetobscuronet
```